.github/CODEOWNERS

```diff
@@ -125,11 +125,14 @@
 /sdk/tables/                                                         @annatisch @YalinLi0312
 
 # PRLabel: %Media
-/sdk/media/                                                          @naiteeks @bennage @giakas
+/sdk/media/                                                          @naiteeks @giakas
 
 # PRLabel: %Container Registry
 /sdk/containerregistry/                                              @YalinLi0312
 
+# PRLabel: %VideoAnalyzer
```

Comment:
@chidozieononiwu how did you know to add this entry?

Reply:
I reached out to @hivyas  who confirmed that I can add that.